coalib/results/SourceRange.py

```diff
@@ -126,6 +126,27 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Default __str__ representation -
+            Filename: FROM start_line:start_coloumn TO end_line:end_coloumn
+        If whole line is affected -
+            Filename: FROM start_line TO end_line
+        If whole file is affected -
+            Filename
+        """
+        if self.start.line is None and self.end.line is None:
+            return self.start.file
+
+        from_str = ': FROM ' + str(self.start.line)
+        to_str = ' TO ' + str(self.end.line)
+
+        if self.start.column is None and self.end.column is None:
+            return self.start.file + from_str + to_str
+
+        return (self.start.file + from_str + ':' + str(self.start.line) +
+                to_str + ':' + str(self.end.line))
```

Comment:
1. you need to write a testcase for this in `SourceRangeTest`
2. I think it's a bit simpler to read if you do this:
   ```python
   if line is None or column is None:
        format_str = '...'
    elif other_condition:
        # choose different format
    elif another_condition:
        ...
    return format_str.format(...)
   ```